model/src/main/java/org/projectnessie/api/ConfigApi.java

```diff
@@ -26,9 +25,7 @@
 import org.eclipse.microprofile.openapi.annotations.responses.APIResponses;
 import org.projectnessie.model.NessieConfiguration;
 
-@Path("config")
```

Comment:
Why only move the REST and OpenAPI annotations from the interfaces `ConfigApi`, `ContentsApi` and `TreeApi` but not the REST and OpenAPI annotations from the methods/function + parameters?

Reply:
see PR description for this

> all validation-specific (and HTTP-specific) annotations is still in the ConfigApi / ContentsApi / TreeApi interfaces because one can't add validation constraints to subclasses of ConfigApi / ContentsApi / TreeApi (the idea was that HttpConfigApi / HttpContentsApi / HttpTreeApi could define all those annotations but the Bean validation Spec doesn't allow that)

Reply:
Any other solution for this maybe?

Reply:
Having REST + OpenAPI definitions "leak" into grpc doesn't feel good either.

Reply:
oh I absolutely agree with you that this doesn't feel good, but I haven't found a good solution for this yet.